prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -214,9 +214,12 @@ class HttpUtilities {
             asyncProcessMode: Boolean = false,
             key: String? = null,
             option: Options? = null,
+            payloadName: String? = null
         ): Pair<Int, String> {
             if (!file.exists()) error("Unable to find file ${file.absolutePath}")
-            return postReportBytes(environment, file.readBytes(), sendingOrgClient, key, option, asyncProcessMode)
+            return postReportBytes(
```

Comment:
Allow the TestReportStream tests to send a payloadName.   Nice for testing, and very useful when browsing the data.